### PR TITLE
Review PR for BXMSDOC-5426 Integrating Red Hat CodeReady Studio with DM/PAM 7.7

### PR DESCRIPTION
**Pull request by michelehaglund**
Updated file names for 7.7

Original jira: https://issues.redhat.com/browse/BXMSDOC-5426
Rendered output:
DM: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-5426DM/
PAM: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-5426/

### CODE REVIEW

File: doc-content/enterprise-only/integration/codeready-studio-install-proc.adoc [hunk 1]
```diff
@@ -1,25 +1,25 @@
 [id='codeready-studio-install-proc']
-= Installing Red Hat CodeReady Studio
+= Installing {CODEREADY_STUDIO}
 
-Red Hat CodeReady Studio is an integrated development environment (IDE) based on Eclipse. It integrates tooling and run time components by combining Eclipse, Eclipse Tooling, and {EAP_LONG}. Red Hat CodeReady Studio provides plug-ins with tools and interfaces for {PRODUCT}.
+{CODEREADY_STUDIO} is an integrated development environment (IDE) based on Eclipse. It integrates tooling and run time components by combining Eclipse, Eclipse Tooling, and {EAP_LONG}. {CODEREADY_STUDIO} provides plug-ins with tools and interfaces for {PRODUCT}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/integration/codeready-studio-install-proc.adoc]: runtime

File: doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
 
 .Procedure
 . Download the {DECISION_ENGINE}:
 .. Log in to the https://access.redhat.com[Red Hat Customer Portal].
 .. Click *DOWNLOADS* at the top of the page.
 .. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *{PRODUCT}*.
 .. On the *Software Downloads* page, download *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons* (`{PRODUCT_FILE}-add-ons.zip`).
-.. Extract `{PRODUCT_FILE}-add-ons.zip` and then extract the Drools runtime environment JAR files located in `{PRODUCT_FILE}-add-ons/{PRODUCT_FILE}-{URL_COMPONENT_DECISION_ENGINE}.zip`.
-. From the Red Hat CodeReady Studio menu, click *Window* -> *Preferences*.
+.. Extract `{PRODUCT_FILE}-add-ons.zip` and then extract the Drools runtime environment JAR files located in
```
Comment (emmurphy1) [inline doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc]: Drools is correct for DM and PAM?

File: doc-content/enterprise-only/integration/codeready-studio-install-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/integration/codeready-studio-install-proc.adoc]: Fixed. Thank you.

File: doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc]: Checking with QE. Thank you.

Comment (michelehaglund) [reply doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc]: I confirmed both files in the CR1 RHDM and RHPAM builds.

**Pull request merged**